tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -595,7 +595,7 @@ Default | CLI Override | API Override
 Use single quotes instead of double quotes.
 
 Notes:
-* Quotes in JSX will always be double and ignore this setting.
+* JSX will always be double quotes and ignore this setting. To use single quotes in JSX use \`jsxSingleQuote\`
```

Comment:
Not sure if this snapshot really should have been updated.

Reply:
I'm not sure either, will revert